### PR TITLE
update product screenshot to reflect new version

### PR DESCRIPTION
**Pull request by sophiehe**
Please provide the following information to help us approve your PR. Failure to provide this
information may result in your PR being closed without comment.

- [ ] Clearly explain what your PR is trying to accomplish. A link to an issue is best
- [ ] Provide us detailed instructions on how we can test your PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Learn more on how to [configure your project](https://docs.reactioncommerce.com/
 - [Reaction Commerce YouTube videos](https://www.youtube.com/user/reactioncommerce/videos)
 
 ## Join the community calls
-- [Reaction Community calls](http://getrxn.io/2rcCal): Join our biweekly community calls every other Wednesday at 7AM PST/10AM EST. 
+- [Reaction community calls](http://getrxn.io/2rcCal): Join our biweekly community calls every other Wednesday at 7AM PST/10AM EST.
```
Comment (zenweasel) [inline README.md]: Should we change this to reflect the rotating schedule of 7/9 am?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 [Reaction](http://reactioncommerce.com) is an event-driven, real-time reactive commerce platform built with JavaScript (ES6). It plays nicely with npm, Docker, and React.
 
-![Reaction v.1.0.0](https://raw.githubusercontent.com/reactioncommerce/reaction-docs/master/assets/rc-desktop.png)
+![Reaction v.1.0.0](https://raw.githubusercontent.com/reactioncommerce/reaction-docs/master/assets/Reaction-Commerce-Illustration-BG-800px.png)
```
Comment (zenweasel) [inline README.md]: Should we update the version # as well?

File: README.md (hunk 1 above)
Comment (spencern) [reply README.md]: @sophiehe @zenweasel who's making this change?

Comment (zenweasel) [reply README.md]: I feel like this should be the PR creator? @sophiehe? Or we can just not do it at 9am any more/ever which I am also in favor of.

Comment (machikoyasuda) [reply README.md]: `7AM PST/10AM EST/3PM GMT`

**Comment by sophiehe** [comment]
@spencern this PR looks good to go. let's get it merged for the next release

**Pull request merged**